### ISSUE TITLE
partial screen refresh with ATI hardware on Linux

### ISSUE THREAD

**Issue by jdtournier** [MRtrix3/mrtrix3#285]
The screen refresh is often incomplete when running MRView on ATI hardware - only parts of the display are actually updated, and different bits get updated at different times. I've now seen it with Qt4 & Qt5, and on Ubuntu 12.04 & 14.04. I'll try running a debug version on one of those systems, see if there's anything obvious.

**Comment by jdtournier** [comment]
I think I finally got somewhere with this. This was an issue since the volume render really wasn't working at all properly on ATI hardware. Everything seems to work fine when using a non-core profile. So the issue has sprung up since we switched to core profile by default not long ago. I also had to fix issues with the config file being loaded _after_ the core profile had been set, making the `NeedOpenGLCoreProfile` config entry useless... 

Anyway, at least we can now tell ATI users who are having issues to try using a non-core profile. 

As to why we have this issue, I'm not sure. Either it's a bug in the ATI fglrx drivers (unlikely), a bug in Qt's handling of buffer swapping when using a core profile (not that unlikely, there's been a lot of changes in their OpenGL handling recently), or MRView is doing something that isn't completely compliant with the 3.3 core profile (again, not unlikely). Problem is there's no errors reported by the OpenGL driver, so no clue as to where the issue might be. 

I'm going to leave this issue open, since I don't think it's actually fixed - I reckon the most likely thing is there is something not right in our OpenGL handling. But at least we have a workaround, and hopefully at some point this will magically fix itself...
